components/style/properties/declaration_block.rs

```diff
@@ -481,16 +481,14 @@ pub fn append_serialization<'a, W, I, N>(dest: &mut W,
 
     // for normal parsed values, add a space between key: and value
     match &appendable_value {
-        &AppendableValue::Css(_) => {
-            try!(write!(dest, " "))
-        },
         &AppendableValue::Declaration(decl) => {
             if !decl.value_is_unparsed() {
                 // for normal parsed values, add a space between key: and value
                 try!(write!(dest, " "));
             }
          },
-         &AppendableValue::DeclarationsForShorthand(..) => try!(write!(dest, " "))
+         &AppendableValue::DeclarationsForShorthand(..) => try!(write!(dest, " ")),
+        _ => {},
```

Comment:
Instead of a wildcard, let's do `AppendableValue::Css(..) => {}`, so if new `AppendableValue` kinds are added it's not forgotten here.